### PR TITLE
Updated readme file for the Operation-Agreement Tangle Community Treasury

### PR DESCRIPTION
**Pull request by Deep-Sea-888**
# Description of change

Updated and added content to the readme of the Tangle Treasury Operating Agreement Github page.

## Type of change

- Added content.

## Change checklist

- [x ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes
- [ ] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

**Comment by Deep-Sea-888** [comment]
@Phyloiota - This is initial content for the readme. Should be and will be continued to add update the readme with better content. I just wanted to get something in there to look presentable.

File: README.md [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
 
-<!-- LICENSE -->
-## License
+---
 
-Distributed under the Apache License. See `LICENSE` for more information.
+## About
 
-<p align="right">(<a href="#top">back to top</a>)</p>
+This repository contains governance documents that were collectively created by the IOTA and Shimmer community, with the support of the IOTA foundation. The community voted to adopt these frameworks as the operating processes and procedures for conducting Shimmer governance and managing the Shimmer community treasury.
```
Comment (Phyloiota) [inline README.md:21]:
```suggestion
This repository contains governance documents that were collectively created by the IOTA and Shimmer community with the support of the IOTA Foundation. The community voted to adopt these frameworks as the operating processes and procedures for conducting Shimmer governance and managing the Shimmer community treasury.
```

File: README.md [hunk 2]
```diff
@@ -1,33 +1,45 @@
...
 
+  - [Exhibit B](https://github.com/Tangle-Community-Treasury-DAO/Operation-Agreement/blob/main/Exhibit%20B%20-%20V1.md): Shimmer Governance Framework
+  - [Exhibit C](https://github.com/Tangle-Community-Treasury-DAO/Operation-Agreement/blob/main/Exhibit%20C%20-%20V1.md): Shimmer Community Grant Committee Framework
```
Comment (Phyloiota) [inline README.md:24]:
```suggestion
  - [Exhibit C](https://github.com/Tangle-Community-Treasury-DAO/Operation-Agreement/blob/main/Exhibit%20C%20-%20V1.md): Shimmer Community Grant Committee Specification
```

Comment (Phyloiota) [reply README.md:24]: I think we should call it specification, Thats how we did it most of the time WDYT? So there is a framework and a specification, maybe avoids confusion

File: README.md [hunk 3]
```diff
@@ -1,33 +1,45 @@
...
 
+These exhibits are constitutive documents that establish the Tangle DAO LLC operating agreement for the Shimmer community treasury LLC (Tangle DAO) formed in the Republic of the Marshall Islands.
```
Comment (Phyloiota) [inline README.md:26]: Check with Paolo if "constittutive" is the legally correct term

Comment (Phyloiota) [inline README.md:26]:
```suggestion
These exhibits are constitutive documents that establish the Tangle Treasury DAO LLC operating agreement for the Shimmer community treasury Grant Committee) formed in the Republic of the Marshall Islands.
```

Comment (Phyloiota) [reply README.md:26]: Here we need to use the legally correct terms. The LLC that will be established shall be named Tangle Treasury DAO LLC (afaik). This LLC operated the Shimmer Community Treasury Grant Committee as it was called in the proposals and voted by the community under this name.
Also check if these terms should be capitalized

File: README.md [hunk 4]
```diff
@@ -1,33 +1,45 @@
...
 
-<!-- CONTACT -->
-## Contact
+## Contributions
 
-Your Name - [@twitter_handle](https://twitter.com/twitter_handle) - email@email_client.com
+The community welcomes contributions and recommendations to further develop the governance and treasury frameworks. However, any changes that alter the intent of Exhibit B and Exhibit C must be completed by following the procedures outlined in Exhibit B. After a successful phase III community vote, a pull request can be submitted to update the exhibits in accordance with the community's agreed consensus.
 
-Project Link: [https://github.com/iotaledger/template](https://github.com/iotaledger/template)
+Minor revisions to this legal document, such as correcting typographical errors or formatting inconsistencies, may be made without community approval. However, any changes that affect the substantive meaning or purpose of the document must be approved by the appropriate parties.
```
Comment (Phyloiota) [inline README.md:32]: Maybe "appropriate parties" is also not a term that should be used legally better to refer to "the community via governance vote" or similar terminology

File: README.md [hunk 5]
```diff
@@ -1,33 +1,45 @@
...
 
-<p align="right">(<a href="#top">back to top</a>)</p>
+## Joining the discussion
 
+If you wish to join the community to advance decentralized governance on the Shimmer and IOTA networks, weekly meetings and daily discussions are held within the IOTA & Shimmer Discord channel, specifically in the [#governance-discussion channel](https://discordapp.com/channels/397872799483428865/839927402242637834). Additionally, all treasury information and access to submitting grant proposals to the community treasury can be found at [TangleTreasury.org](https://www.tangletreasury.org).
+    
+If you would like to join the community to advance decentralized governance on the Shimmer and IOTA networks, there are weekly meetings and daily discussions within the IOTA & Shimmer discord found in the [#governance-discussion thread](https://discordapp.com/channels/397872799483428865/839927402242637834).
```
Comment (Phyloiota) [inline README.md:38]:
```suggestion
If you would like to join the community to advance decentralized governance on the Shimmer and IOTA networks, there are weekly meetings and daily discussions within the IOTA & Shimmer discord found in the [#governance-discussion channel](https://discordapp.com/channels/397872799483428865/839927402242637834). 
```

Comment (Phyloiota) [reply README.md:38]: I would suggest to also include information about the Governance forum

File: README.md [hunk 6]
```diff
@@ -1,33 +1,45 @@
+![Tangle Treasury Logo](https://github.com/Deep-Sea-888/Operation-Agreement/blob/main/smr_community_treasury_white_large.png?raw=true)
 
+<h2 align="center">Shimmer Community Framework</h2>
 
+<p align="center">
+  <a href="https://discord.iota.org/" style="text-decoration:none;"><img src="https://img.shields.io/badge/Discord-9cf.svg?logo=discord" alt="Discord"></a>
+  <a href="https://wiki.iota.org/shimmer/learn/governance/shimmer-governance-intro" style="text-decoration:none;"><img src="https://img.shields.io/badge/Wikipedia-Governance-blue" </a>
+  <a href="https://tangletreasury.org/" style="text-decoration:none;"><img src=https://img.shields.io/badge/TangleTreasury-.org-blue"
+</p>
```
Comment (Phyloiota) [inline README.md:9]: Potentially add the link to the Governance Forum

**Review by Phyloiota — CHANGES_REQUESTED** [review]
See my comments

File: README.md (hunk 3 above)
Comment (Deep-Sea-888) [reply README.md:26]: Sent it to him to get confirmation.

File: README.md (hunk 4 above)
Comment (Deep-Sea-888) [reply README.md:32]: Good point.

**Comment by Deep-Sea-888** [comment]
I made some changed and I think I submitted for merge. Let me know if you see on your end. 

One change would be that you should see is the button link to Govern.IOTA.org

**Review by Phyloiota — APPROVED** [review]

**Pull request merged**